### ISSUE TITLE
dotcache - Remove context caching

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#24075]
### Problem Statement

When updating the #dotcache directive, we added the ability to cache not only the text content in the cache, but also what was set in the context in the #dotcache block.

For customers who have complex #dotcache blocks, this can be problematic as it can result in variables set one way on a page to be used on other pages that share the same cache key.

### Steps to Reproduce

Add this to a page:
```
#dotcache("testing", 300)
    cached date is $date.date
    #set($thisIsATest = "Cache is setting context!")
#end
$thisIsATest
```

You will see output:
```
cached date is Thu Feb 09 20:08:53 UTC 2023
Cache is setting context!
```

The `$thisIsATest` value is also being cached.

### Acceptance Criteria

You should not see `Cache is setting context!` message any longer.

### dotCMS Version

23.01

### Proposed Objective

Reliability

### Proposed Priority

Priority 2 - Important

**Comment by erickgonzalez** [comment]
The code that broke this was included in 23.01.

**Comment by bryanboza** [comment]
Fixed, tested on release-23.03 // Docker // FF
